package.json

```diff
@@ -34,25 +34,24 @@
   "authors": [
     "Jason Miller <jasonmiller@synacor.com>"
   ],
-  "license": "BSD 3-Clause",
+  "license": "BSD-3-Clause",
   "files": [
     "src",
     "dist"
   ],
   "eslintConfig": {
-    "extends": "eslint-config-synacor"
+    "extends": "eslint-config-synacor",
+    "rules": {
+      "guard-for-in": 0
```

Comment:
why did you add this here instead of just adding a comment to ignore the rule where it was safe to ignore it?

Reply:
I believe it was in two places. It's a slightly overzealous rule for library code.